openedx/core/djangoapps/content/block_structure/manager.py

```diff
@@ -92,9 +92,9 @@ def get_collected(self):
                 from each registered transformer.
         """
         try:
-            block_structure = BlockStructureFactory.create_from_store(
+            block_structure = BlockStructureFactory.create_from_modulestore(
```

Comment:
Sorry I missed this originally, but why did we make this change?

Reply:
I'm asking because it seems like there are a lot of query count jumps, and it seems like this would be the most likely culprit for why that happened.

Reply:
I assume block structures are stored in modulestore so I make this change.I do't make this change I am getting the error "Blockstructure not found".